app/scripts/modules/core/src/presentation/colors.module.css

```diff
@@ -0,0 +1,7 @@
+@value white: #fff;
```

Comment:
I definitely should've left a note about this stuff (same goes for `breakpoints` and `layers`) — my intent here is to experiment with what it would be like to actually use CSS Modules for the full scope of things we do all the way from shared variables/building blocks up to actual components on the screen, so I am in some cases duplicating things or responsibilities that are already in the `styleguide` module. Once we align on a direction for this experimental stuff, the end-goal is to fold this set of variables into whatever becomes the 'paved path' way (whether that's anything like either of the two things we have now remains to be seen).